recipe/meta.yaml

```diff
@@ -11,6 +11,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
+  skip: True  # [not py37]
```

Comment:
You'll probably want to remove this and re-render again.

Reply:
@ocefpaf For the current ObsPy version we have all packages built already besides the new py37. I just wanted to avoid having new redundant packages for all Python versions. But if you prefer I can just have all Py versions built in this PR regardless?

(I intended to remove that line for the next new ObsPy version build)